ITEST/itest.security/src/org/openl/rules/ruleservice/spring/HackTestSecurityValidator.java

```diff
@@ -0,0 +1,25 @@
+package org.openl.rules.ruleservice.spring;
+
+import org.openl.rules.ruleservice.servlet.SecurityValidationResult;
+import org.openl.rules.ruleservice.servlet.SecurityValidator;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Component
+public class HackTestSecurityValidator implements SecurityValidator {
+
+    public boolean supports(HttpServletRequest request) {
+        String hackHeader = request.getHeader("hack-header");
+        return hackHeader != null && !hackHeader.isEmpty();
+    }
+
+    @Override
+    public SecurityValidationResult validate(HttpServletRequest request) {
+        if (!supports(request)) {
+            throw new RuntimeException("Can't validate request");
+        }
+        return new SecurityValidationResult(request, this).setValidated(true);
```

Comment:
What is the purpose of `SecurityValidationResult`? It just keeps two references and nothing more. Why don't you return just boolean?

Reply:
If so, why you can't simply do it in the place where you execute these validators?
As a developer, I don't really want to know about this wrapper because it has any sense to me and means nothing. It's just like a noize